src/Classes/Controllers/GetEventController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+
+namespace Portal\Controllers;
+
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+class GetEventController
```

Comment:
`GetEventsController` would make sense because it always gets all of the events - someone may want to add a `GetEventController` later to get a single event